src/Domain/IssuedInvoice.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+namespace DDDWorkshop\Domain;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+class IssuedInvoice
+{
+    /** @var UuidInterface */
+    private $id;
+
+    /** @var Buyer */
+    private $buyer;
+
+    /** @var Items */
+    private $items;
+
+    /** @var DateOfIssue */
+    private $dateOfIssue;
+
+    /** @var DateOfPayment */
+    private $dateOfPayment;
+
+    /** @var mixed due to phpStan */
+    private $number;
+
+
+    /**
+     * @param Buyer $buyer
+     * @param Items $items
+     * @param DateOfIssue $dateOfIssue
+     * @param DateOfPayment $dateOfPayment
+     * @param Series $series
+     * @param RegistrNespolehlivychPlatcu $registrNespolehlivychPlatcu
+     * @throws \Exception
+     */
+    public function __construct(
+        Buyer $buyer,
+        Items $items,
+        DateOfIssue $dateOfIssue,
+        DateOfPayment $dateOfPayment,
+        Series $series,
+        RegistrNespolehlivychPlatcu $registrNespolehlivychPlatcu
+    ) {
+        if (! $registrNespolehlivychPlatcu->check($buyer)) {
+            throw new \DDDWorkshop\Domain\Exceptions\NespolehlivyPlatceException();
+        }
+        if ($dateOfIssue > $dateOfPayment) {
```

Comment:
you don't have a test for this, but this comparation shouldn't work.

I recommend to rewrite it to `$dateOfIssue->checkBeforeOrSame($dateOfPayment);`
and also that this check method throws an exeption